app/code/Magento/Ui/TemplateEngine/Xhtml/Result.php

```diff
@@ -124,7 +124,10 @@ public function __toString()
             $this->appendLayoutConfiguration();
             $result = $this->compiler->postprocessing($this->template->__toString());
         } catch (\Throwable $e) {
-            $this->logger->critical($e);
+            $this->logger->critical("Error while rendering template: {message}", [
```

Comment:
Hi @marcusirgens. Could you provide more information, please, why do we need `{message}` part here? It does not seem to be replaced by the real message somehow in the result.

Reply:
Hi, @rogyar! The [PSR-3 spec ](https://www.php-fig.org/psr/psr-3/#12-message), which Magento's logger uses, suggests the use of placeholders. https://github.com/magento/magento2/pull/29796 adds a processor to the default Magento 2 log stream that populates these placeholders in the default FS logs. :)

Reply:
It makes sense, thanks for the explanation